src/Controls/tests/Xaml.UnitTests/MSBuild/MSBuildTests.cs

```diff
@@ -388,7 +388,7 @@ public void DesignTimeBuild()
 			var xamlCStamp = IOPath.Combine(intermediateDirectory, "XamlC.stamp");
 
 			//The assembly should not be compiled
-			AssertDoesNotExist(assembly);
+			//AssertDoesNotExist(assembly);
```

Comment:
Why is this assert commented out?

Reply:
cause there are now multiple source gen contributing to an assembly being created